src/libs/state.ts

```diff
@@ -259,9 +259,9 @@ export const workspacesStore = atom<WorkspaceWrapper[]>([]);
 
 export const rerunFailuresStatus = atom<unknown>(undefined);
 
-export const errorNotifiedRuntimes = atom<unknown[]>([]);
+export const errorNotifiedRuntimes = atom<number[]>([]);
```

Comment:
Need types here because TypeScript wouldn't allow appending a number or string to a list of `unknown`.